components/status_dropdown/status_dropdown.tsx

```diff
@@ -328,6 +328,7 @@ export default class StatusDropdown extends React.PureComponent <Props, State> {
                     action: index === 4 ? () => setCustomTimedDnd() : () => setDnd(index),
                 } as any;
             }));
+        console.log('dndSubMenuItems', dndSubMenuItems);
```

Comment:
Are both changes on this file for debugging?

Reply:
Thanks, Done